### PR TITLE
Aaronhmiller patch 1

### PR DESCRIPTION
**Pull request by aaronhmiller**
**NOTE**: Please read the CONTRIBUTING.md guidelines before submitting your patch,
and ensure you followed them all:

https://github.com/Kong/getkong.org/blob/master/CONTRIBUTING.md

### Summary

SUMMARY_GOES_HERE

### Full changelog

* [Edited docs to ...]
* [Added cool new feature ...]
* ...

### Issues resolved

Fix #XXX

### Checklist:

- [x] Spellchecked my documentation
- [x] Documentation 
- [x] Ready to be merged

### CODE REVIEW

**Comment by aaronhmiller** [comment]
We're getting feedback from the field asking why "Create Developer Account" button doesn't work. Positioning in doc more accurately so we needn't verbally explain each time and make users think something is wrong on first touch.

File: app/docs/enterprise/0.31-x/developer-portal/introduction.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+title: Welcome to the Kong Developer Portal Preview
 ---
 
-# [Welcome to the Kong Dev Portal](#welcome-to-the-kong-dev-portal)
+# [Welcome to the Kong Dev Portal Preview](#welcome-to-the-kong-dev-portal)
 
 ![alt text](https://konghq.com/wp-content/uploads/2018/03/screen-home.png "Welcome to the Kong Dev Portal")
 
-Thanks for installing or upgrading Kong Enterprise Edition. This document orients you to Kong's built-in Dev Portal functionality and gets you started with customizing your Kong Dev Portal.
+Thank you for installing Kong Enterprise Edition. This document orients you to Kong's built-in Dev Portal functionality and provides you a preview into it so you can familiarize yourself with its promising capabilities and get started on customizing it to match your needs.
```
Comment (darrenjennings) [inline app/docs/enterprise/0.31-x/developer-portal/introduction.md]: Suggestion: "and provides you a preview ~~into it~~ so you can familiarize yourself"

File: app/docs/enterprise/0.31-x/kong-implementation-checklist.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ title: Kong Implementation Checklist
 
 ## Production System Requirements
 
-- For Kong prefer CPU optimized instances. We recommend at least 4 CPU cores, 16GB of RAM and 24GB of disk space. Kong performs better on a system with multiple cores. In an AWS environment, we recommend running c 3.2xlarge instances in production that guarantee enough cores (8) and memory (16GB), with a local storage instead of EBS to improve the performance.
+- For Kong prefer CPU optimized instances. We recommend at least 4 CPU cores, 16GB of RAM and 24GB of disk space. Kong performs better on a system with multiple cores. In an AWS environment, we recommend running c4.2xlarge instances in production that guarantee enough cores (8) and memory (16GB), with a local storage instead of EBS to improve the performance.
```
Comment (rainest) [inline app/docs/enterprise/0.31-x/kong-implementation-checklist.md]: Should strike the part about local storage, as current compute-optimized instance sizes are EBS-only.

Comment (aaronhmiller) [reply app/docs/enterprise/0.31-x/kong-implementation-checklist.md]: Good catch. Thank you for the careful read and research Travis! Updating.

**Comment by aaronhmiller** [comment]
Any blockers to merging this in?

**Pull request merged**